CHANGELOG.md

```diff
@@ -55,6 +55,8 @@ Base64 encoded transactions.
 * [\#164](https://github.com/cosmos/gaia/pull/164) [\#212](https://github.com/cosmos/gaia/pull/212) The LevelDB-based
 keybase implementation has been replaced with a 99 designs Keyring library-backed implementation. Keys created and stored
 with previous `gaia` releases need to be migrated through the `gaiacli keys migrate` command.
+* [\#349](https://github.com/cosmos/gaia/pull/349) The new implementation of the Keyring from SDK does not allow
```

Comment:
CHANGELOG might need some extra love. I'll take care of it when this is ready :+1:

Reply:
As a long term user/developer of the SDK/Gaia, I don't know what this means.

Reply:
There is no need for a CHANGELOG addition. The migration to the new keyring is already mentioned. Hence resolving this.